### ISSUE TITLE
Remove use of outline: none to support high contrast colors mode

### ISSUE THREAD

**Issue by clhenrick** [clhenrick/clhenrick.io#64]
Instead use `outline: transparent` so that forced colors modes may still apply an outline color to elements.

**Comment by clhenrick** [comment]
Should be fixed in #63
